rust/kcl-lib/src/execution/geometry.rs

```diff
@@ -1293,6 +1337,14 @@ impl Path {
                 // TODO: Call engine utils to figure this out.
                 linear_distance(&self.get_base().from, &self.get_base().to)
             }
+            Self::Ellipse { .. } => {
+                //TODO: fix me
+                10.0
+            }
+            Self::Conic { .. } => {
+                //TODO: fix me
+                10.0
```

Comment:
I believe the segment length is used in a couple places:

1. In sketch overlays, we display the length to the user.
2. KCL functions like `segLen()` expose it to KCL.

For overlays, we can make special handling that doesn't show it for these segment types.

For `segLen()`, we can return a runtime error saying that it's not supported. We can change the return type of `length()` to be an Option.

Reply:
I made the runtime error for `segLen()` here: 2adcf393a8102502b42620237478f16e0599861b. For the frontend stuff, it might be helpful to get the engine merged first.